cobra/config.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2014 Novo Nordisk Foundation Center for Biosustainability, DTU.
```

Comment:
just an oversight, cleaning out the apache mentions, no intention to change licenses anywhere